components/brave_new_tab_ui/components/default/braveToday/customize/ChannelCard.tsx

```diff
@@ -20,30 +26,27 @@ const Container = styled(Flex) <{ backgroundColor: string }>`
   color: white;
   position: relative;
 
-  :hover {
-    opacity: 0.8;
+  &[data-channel-card-is-followed=true] {
```

Comment:
Okay, so I found this a bit weird. When I click on the button, it sets the `:focus-within` pseudo element so the button stays visible, even when I move the mouse outside. Then, when I click on something else, the button disappears.

Reply:
ohh right

Reply:
Fixed with `:has(:focus-visible)`